### PR TITLE
wallet UI to update and improvement transaction display

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
improment for wallet ui:
- sort token by usd
- add more detail in transaction tab of pool/farm transactions

### CODE REVIEW

File: src/components/Popups/TransactionPopup.tsx [hunk 1]
```diff
@@ -57,14 +57,18 @@ const summaryClaim = (txs: TransactionDetails) => {
 }
 
 const summaryStakeUnstakeFarm = (txs: TransactionDetails) => {
-  const { summary = 'liquidity' } = (txs.extraInfo || {}) as TransactionExtraBaseInfo
-  return txs.type === TRANSACTION_TYPE.STAKE ? `Stake ${summary} into farm` : `Unstake ${summary} from farm`
+  return txs.type === TRANSACTION_TYPE.STAKE ? `Stake liquidity into farm` : `Unstake liquidity from farm`
```
Comment (namgold) [inline src/components/Popups/TransactionPopup.tsx:60]:
```suggestion
  return txs.type === TRANSACTION_TYPE.STAKE ? t`Stake liquidity into farm` : t`Unstake liquidity from farm`
```

File: src/components/WalletPopup/SendToken/index.tsx [hunk 2]
```diff
@@ -232,18 +264,14 @@ export default function SendToken({
...
-                : '-'}{' '}
-            </Label>
-          </RowBetween>
-        )}
+        <RowBetween>
+          <Label>
+            <Trans>Gas Fee</Trans>
+          </Label>
+          <Label color={theme.text}>
+            {estimateGas && usdPrice ? `~ ${formattedNum((estimateGas * usdPrice).toString(), true)} ` : '-'}
```
Comment (viet-nv) [inline src/components/WalletPopup/SendToken/index.tsx:272]: `estimateGas * usdPrice` => this should be price of native token right? not usdPrice of token in

File: src/components/RewardTokenPrices/index.tsx [hunk 3]
```diff
@@ -148,7 +148,7 @@ const RewardTokenPrices = ({ style = {}, rewardTokens }: { style?: React.CSSProp
           {rewardTokens.map((token, index) => {
             return (
               <TokenWrapper
-                key={token.wrapped.address}
+                key={token.wrapped.address + index}
```
Comment (hungdoansy) [inline src/components/RewardTokenPrices/index.tsx:151]:
```suggestion
                key={index}
```

File: src/components/AddressInputPanel/index.tsx [hunk 4]
```diff
@@ -72,25 +72,31 @@ const DropdownIcon = styled(DropdownSVG)<{ open: boolean }>`
...
-  placeholder,
-  icon,
-  disabled = false,
-  style = {},
-  className,
-}: {
+type Props = {
   onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  onFocus?: (e: ChangeEvent<HTMLInputElement>) => void
+  onBlur?: (e: ChangeEvent<HTMLInputElement>) => void
```
Comment (hungdoansy) [inline src/components/AddressInputPanel/index.tsx:78]:
```suggestion
  & Pick<DOMAttributes< HTMLInputElement>, 'onBlur' | 'onFocus' | 'onChange'>
```

File: src/components/WalletPopup/SendToken/index.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/components/WalletPopup/SendToken/index.tsx]: nice catch. updated

File: src/components/AddressInputPanel/index.tsx (hunk 4 above)
Comment (nguyenhoaidanh) [reply src/components/AddressInputPanel/index.tsx:78]: meet this issues when follow your suggestion
image

Comment (nguyenhoaidanh) [reply src/components/AddressInputPanel/index.tsx]: nice

**Review by viet-nv — APPROVED** [review]

**Pull request merged**